docker/test-images/zipkin-cassandra/install.sh

```diff
@@ -63,6 +73,12 @@ mvn -q --batch-mode -DoutputDirectory=lib \
     org.apache.maven.plugins:maven-dependency-plugin:3.8.1:copy-dependencies
 rm pom.xml
 
+# Patch Cassandra 4.1's ObjectSizes.java to work with newer jamm.
+wget --random-wait --tries=5 -qO ObjectSizes.java \
+  https://raw.githubusercontent.com/apache/cassandra/refs/tags/cassandra-5.0.3/src/java/org/apache/cassandra/utils/ObjectSizes.java
+sed -i 's/sizeOnHeapExcludingDataOf/sizeOnHeapExcludingData/g' ObjectSizes.java
+javac -cp 'lib/*' -d classes ObjectSizes.java
```

Comment:
this works as some time in the past we were already adding classes. So, "classes" is already in the run scripts else startup would fail.